tools/db_stress.cc

```diff
@@ -1023,20 +1023,16 @@ class DbStressListener : public EventListener {
         column_families_(column_families) {}
   virtual ~DbStressListener() {}
 #ifndef ROCKSDB_LITE
-  virtual void OnFlushCompleted(DB* db, const FlushJobInfo& info) override {
-    assert(db);
```

Comment:
instead of removing the assert, can we use unused attributed?

Reply:
@ajkr suggested these assert are useless since for db_stress they will always be true

Reply:
yeah it seemed strange to me that we're testing whether `OnFlushCompleted` is passed the right arguments in our stress test, so I suggested deleting those checks rather than maintaining them.